server/routes.js

```diff
@@ -378,6 +379,29 @@
         }
     });
 
+    var storage = multer.diskStorage({
+        destination: function (req,file, callback){
+            callback(null, String(req.body.upload_path)) 
+        },
+        filename: function(req,file,callback){
+            callback(null, file.originalname)
+        }
+    });
+
+    var upload = multer({storage: storage }).array('file',3);
```

Comment:
These two vars storage and upload.

Does it have to be global ? 

If yes can you move it to the tools.js or create another file with it ? 
If not include them into the router function.